### ISSUE TITLE
Problems installing okapi-em plugin in Mac M1 pro

### ISSUE THREAD

**Issue by adosnsantos** [rosalindfranklininstitute/okapi-em#6]
Hi, great work developing this tool. Looks really cool and I am trying to install this plugin in Napari to use with my cryo-FIB data, but I have encountered a few issues. I wasn't able to install through pip install or through the gui (Plugins>>Install) as it kept getting stuck on 'Preparing metadata (pyproject.toml): started'. In the end I managed to install it using conda install though. But now the plugin does not look like it should. I only have the C-supression and the Slice Alignment options available and in the Slice alignment I have no options at all. Just the align button. Any ideas of how to try to solve the issue would be great!!

Cheers!

<img width="1122" alt="Screenshot 2023-08-01 at 12 57 22" src="https://github.com/rosalindfranklininstitute/okapi-em/assets/127216746/076c4c04-029f-41e2-a81c-2b386bc6987c">
<img width="1122" alt="Screenshot 2023-08-01 at 12 57 44" src="https://github.com/rosalindfranklininstitute/okapi-em/assets/127216746/e8c81b65-26d5-4ca9-a837-9b0f8c5bf0dd">
<img width="1122" alt="Screenshot 2023-08-01 at 12 58 21" src="https://github.com/rosalindfranklininstitute/okapi-em/assets/127216746/35cadcfb-44fe-4e1e-9329-e693b1ebc1ca">

**Comment by perdigao1** [comment]
Indeed. That looks very strange indeed. The automatic tests in github did not report anything wrong with Mac OS

https://github.com/rosalindfranklininstitute/okapi-em/actions/runs/5324924568

If you could provide some additional info like the python version in your environment, it would help.

Another option is to try installing (remove old one first) from command line with >pip install okapi-em

and see if the output is more verbose than in napari. We don't use Mac, so it is difficult to test here.
